rust/kcl-wasm-lib/Cargo.toml

```diff
@@ -53,5 +53,5 @@ wasm-bindgen-test = "0.3.50"
 typed-path = "0.11.0"
 
 # Local development only. Placeholder to speed up development cycle
-#[package.metadata.wasm-pack.profile.release]
-#wasm-opt = false
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
```

Comment:
We should change this back to what it was before. You should be able to get the same (or similar?) faster build using:

```shell
npm run build:wasm:dev
```